Lombiq.Tests.UI/Docs/Configuration.md

```diff
@@ -59,39 +59,36 @@ Recommendations and notes for such configuration:
 
 ### HTML validation configuration
 
-If you want to change some HTML validation rules from only a few specific tests, you can create a custom _.htmlvalidate.json_ file (e.g. _TestName.htmlvalidate.json_). For example:
+If you want to change some HTML validation rules from only a few specific tests, you can create a custom _.htmlvalidate.json_ file (e.g. _TestName.htmlvalidate.json_). This should extend the [default.htmlvalidate.json](../default.htmlvalidate.json) file (which is always copied into the build directory) by setting the value of `"extends"` to a relative path pointing to it. For example:
 
 ```json
 {
-  "extends": [
-    "html-validate:recommended"
-  ],
-
-  "rules": {
-    "attribute-boolean-style": "off",
-    "element-required-attributes": "off",
-    "no-trailing-whitespace": "off",
-    "no-inline-style": "off",
-    "no-implicit-button-type": "off",
-    "wcag/h30": "off",
-    "wcag/h32": "off",
-    "wcag/h36": "off",
-    "wcag/h37": "off",
-    "wcag/h67": "off",
-    "wcag/h71": "off"
-  },
+    "extends": [
+        "./default.htmlvalidate.json"
+    ],
+
+    "rules": {
+        "element-required-attributes": "off",
+        "no-implicit-button-type": "off"
+    },
 
-  "root":  true
+    "root":  true
 }
 ```
 
 Then you can change the configuration to use that:
 
 ```cs
- changeConfiguration: configuration => configuration.HtmlValidationConfiguration.HtmlValidationOptions =
-                configuration.HtmlValidationConfiguration.HtmlValidationOptions
-                    .CloneWith(validationOptions => validationOptions.ConfigPath =
-                        Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "TestName.htmlvalidate.json")));
+changeConfiguration: configuration => configuration.HtmlValidationConfiguration.HtmlValidationOptions =
+    configuration.HtmlValidationConfiguration.HtmlValidationOptions
+        .CloneWith(validationOptions => validationOptions.ConfigPath =
```

Comment:
I don't know, it was already like this in the Configuration.md so I assumed it's necessary.
@DemeSzabolcs Do you remember the reason?

Reply:
FYI I implemented a non-cloning way to set the path under https://github.com/Lombiq/UI-Testing-Toolbox/pull/362 too (will remove it) and nothing bad happened. The config object is unique to the test (ultimately due to `OrchardCoreUITestExecutorConfiguration` being instantiated for each test in `OrchardCoreUITestBase`) so changing its state isn't a problem.

Reply:
> I don't know, it was already like this in the Configuration.md so I assumed it's necessary. @DemeSzabolcs Do you remember the reason or where it came from? Since you added the docs in #333.

I did that based on this:
https://github.com/atata-framework/atata-htmlvalidation/issues/8#issuecomment-1864839487

Reply:
Thanks for the context! In this case it's indeed not necessary because the example was using `HtmlValidationOptions.Default`, but we use `new HtmlValidationOptions { ... }` so it's all unique instances.